jetty-annotations/pom.xml

```diff
@@ -31,8 +31,11 @@
         <configuration>
           <instructions>
             <Import-Package>${osgi.slf4j.import.packages},org.objectweb.asm;version="5",*</Import-Package>
-            <Require-Capability>osgi.serviceloader; filter:="(osgi.serviceloader=javax.servlet.ServletContainerInitializer)";resolution:=optional;cardinality:=multiple, osgi.extender; filter:="(osgi.extender=osgi.serviceloader.processor)";resolution:=optional
+            <Require-Capability>osgi.extender; filter:="(osgi.extender=osgi.serviceloader.registrar)", osgi.serviceloader; filter:="(osgi.serviceloader=javax.servlet.ServletContainerInitializer)";resolution:=optional;cardinality:=multiple, osgi.extender; filter:="(osgi.extender=osgi.serviceloader.processor)";resolution:=optional
```

Comment:
not really related to this PR but there are some discrepancies between poms.
some have single line some have multiple lines.
Perso I prefer multiple lines as it is more human readable (next line after `,` or `;`?)
Maybe we should decide about this?

Reply:
I like the multi-line form as well.
Especially when reviewing changes in diffs.

But, that can be a different PR.